gemfileparser/__init__.py

```diff
@@ -47,9 +47,11 @@ def __init__(self):
     group_block_regex = re.compile(
         r"group[ ]?:[ ]?(?P<groupblock>.*?) do")
     add_dvtdep_regex = re.compile(
-        r".*add_development_dependency (?P<line>.*)")
+        r".*add_development_dependency(?P<line>.*)")
     add_rundep_regex = re.compile(
-        r".*add_runtime_dependency (?P<line>.*)")
+        r".*add_runtime_dependency(?P<line>.*)")
```

Comment:
I know it is tempting to fix formatting here, but keep that for a separate PR that would handle that as well as using black throughout